### DISCUSSION TITLE
Unable to Commit

### DISCUSSION THREAD

**Discussion by souvikb1983** [community/community#29867] [Other Feature Feedback, Questions, & Ideas]
Two team members Bharat Chaudary and Puneet Tripathi unable to commit projects in eclipse.
Please assist them
 https://github.com/cwt-dev/pa_login_v1 as this is not working
![a33dd7c6-fa5c-4f68-83b4-71afce69b65a](https://user-images.githubusercontent.com/86236264/185254193-9c055ee9-5a43-4309-a217-12e8368da097.PNG)

**Comment by verduscos** [comment]
This seems to be an issue with authentication, likely that the users don't have permission to push repo. This [thread ](https://stackoverflow.com/questions/10475726/git-receive-pack-not-permitted-on-push) might help.

**Reply by souvikb1983** [reply]
But all the users in cwt_dev have the same access but still not working so need more help as other folders it's allowing
[wc](https://github.com/cwt-dev/global_hierarchy_project) there is no error.
So same access diferent response
